aspnetcore/blazor/class-libraries.md

```diff
@@ -5,14 +5,14 @@ description: Discover how components can be included in Blazor apps from an exte
 monikerRange: '>= aspnetcore-3.0'
 ms.author: riande
 ms.custom: mvc
-ms.date: 06/14/2019
+ms.date: 06/24/2019
 uid: blazor/class-libraries
 ---
 # ASP.NET Core Razor components class libraries
 
 By [Simon Timms](https://github.com/stimms)
 
-Components can be shared in Razor class libraries across projects. Components can be included from:
+Components can be shared in a [Razor class library (RCL)](xref:razor-pages/ui-class) across projects. A *Razor components class library* can be included from:
```

Comment:
Is a RCL the same thing as a RCCL?  If so, just to avoid possibility of confusion might be best to settle on one name.

Reply:
@danroth27 Drop the *RCCL*? If so, the title should probably go to something like ...

#### Provide ASP.NET Core Razor components in Razor class libraries

Reply:
A Razor class library is a class library that is setup for compiling Razor files (MVC Views, Razor Pages, Components, etc.). I'm fine with referring to Razor class libraries that contain components as Razor component class libraries. Always capitalizing "Razor Class Library" feels a bit excessive to me. I though the trend was towards lower case words?

Reply:
> I thought the trend was towards lower case words?

Just wrt using sentence case in titles and headings. Not with regard to proper nouns that are multiple words. ..... the Eiffel tower? Dan roth? Doctor who!!!! :smile: lol

Now that I re-examine the topic, Rick has it as a proper noun (capitalized), so this should go the same way. It matches the choice to capitalize by the VS devs/managers in the VS UI.

It's a bit of a grey area ... "Razor" in "Razor class library" is an adjective to describe a run-the-mill "class library" that does Razor-y things. That seems accurate. However, a "Razor Class Library" implies that this is a specially named aspect of the technology ... specifically for the purposes that you listed.

Anywho ... hardly matters from a presentation point-of-view. This will finish off the process of using RCL everywhere.

... and another lovely side note (do they ever *END*?! 🛑) ... "Razor Component" is a named, specific thing/aspect of the framework. I think that should be capitalized (i.e., not "Razor component"). I think that's a change that I've been slowly massaging docs into.

Reply:
I've been using "Razor component" instead of "Razor Component" in part to avoid confusion with the old "ASP.NET Core Razor Components" name. But it's also consistent with what other SPA frameworks do. For comparison, would you say "React component" or "React Component"? "Angular component" or "Angular Component"? The React and Angular docs both don't capitalize the "c" in component. Similarly, I don't think we should be capitalizing "class library" except maybe in the case where you are referring to the name of a specific project template. I.e the Razor Class Library template is used to create a Razor class library.

Reply:
Ok ... sounds good. I'll make the updates on the next commit.